CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
 
+project(cryfs)
```

Comment:
I have to admit I'm not 100% sure what a cmake project is. We're currently defining a project for each subdirectory, see for example https://github.com/cryfs/cryfs/blob/4069358b31b56f6c1fa91271aa6d20a211ab347a/src/cryfs/CMakeLists.txt#L1

Does it make sense to still have this here on top level or maybe we don't need it?